src/App.tsx

```diff
@@ -20,9 +23,37 @@ import ItemForm from "./pages/ItemForm";
 import NotFound from "./pages/NotFound";
 import CollectionDetail from "./pages/CollectionDetail";
 
-export const App = () => (
+// Keycloak configuration
+const keycloakInstance = new Keycloak({
+  realm: "stac-realm",
+  url: "http://localhost:8080/",  // Ensure the URL is correct
+  clientId: "stac-admin",
```

Comment:
these should come from environment variables (.env)

Reply:
added environment variables